tests/Cargo.toml

```diff
@@ -25,6 +25,7 @@ aws-config = { version = "1.1.7", features = [
 aws-sdk-kms = { version = "1.37.0", optional = true }
 clap = { workspace = true }
 cynic = { workspace = true }
+ethereum-types = "0.14.1"
```

Comment:
It would be nice if relayer service exported this dependency instead. Maybe behind the `test-helpers` feature. In this case we don't need to include it here. 

Also relayer service could provide the `covnert_to_address` function, and we could use it instead of the `Address` type.

Reply:
Oops, didn't see, seems like you did it :D